stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Channel.kt

```diff
@@ -28,6 +29,10 @@ public fun Channel.isDistinctChannel(): Boolean {
     return cid.contains("!members")
 }
 
+@InternalStreamChatApi
+public fun ChannelData.isDistinctChannel(): Boolean {
```

Comment:
I would move channel type checks to the `client` and make them public but I'm not sure about the naming. Possible names could be:
- isAnonymousChannel
- isDistinctChannel
- isGroupChat

Reply:
I like the `isAnonymousChannel`

Reply:
`ChannelData` is not available there, but I can extend the string.